src/js/letters/containers/LetterList.jsx

```diff
@@ -34,24 +26,19 @@ export class LetterList extends React.Component {
         content = letterContent[letter.letterType] || '';
       }
 
-      let downloadLetterLink;
-      if (this.props.optionsAvailable) {
```

Comment:
So this logic was added in this PR: https://github.com/department-of-veterans-affairs/vets-website/pull/6608. I think the mistake I made was to hide the download button for ALL letters if options aren't available. However, the logic should still exist that if `optionsAvailable === false`, the download button should not appear for the BSL. That's because I checked with CSRA and they said if the options aren't available the letter won't be able to generate, so we might as well hide the button. So I think we need to still account for that here. So the logic should be something like:
```
if (letter.letterType !== LETTER_TYPES.benefitSummary || this.props.optionsAvailable) {
  downloadLetterLink = (<DownloadLetterLink ... />);
}
```

Reply:
That's fine; when deciding how to fix this I reached out to Tim Barto and was told that it was not possible to get a `benefit_summary` letter returned from `v0/letters` and *not* get any available `benefit_information` options returned by `v0/letters/beneficiary`, which is why I removed this bit entirely. It sounds like you've run into this before though, so the check for the BSL at least should go back in.

Reply:
Ah gotcha, I think I had either gotten slightly different info or misinterpreted what he said. Either way, I think this is a good safeguard just in case we get some unexpected data from them!